climpred/metrics.py

```diff
@@ -1729,17 +1729,19 @@ def _brier_score(forecast, verif, **metric_kwargs):
 
     Example:
         >>> def pos(x): return x > 0
-        >>> compute_perfect_model(ds, control, metric='brier_score', func=pos)
+        >>> compute_perfect_model(ds, control, metric='brier_score', logical=pos)
     """
-    if 'func' in metric_kwargs:
-        func = metric_kwargs['func']
+    if 'logical' in metric_kwargs:
+        logical = metric_kwargs['logical']
+        if not callable(logical):
```

Comment:
That actually belonged to the just merged PR...

Reply:
Which PR are you referring to?

Reply:
Also I think in `CHANGELOG` this needs to go under deprecated since it changes a keyword name.

Reply:
added as breaking change